tests/61push/01message-pushed.pl

```diff
@@ -226,3 +226,158 @@
          Future->done(1);
       });
    };
+
+
+sub setup_push
```

Comment:
Could be worth adding a small line of documentation to remark that the returned future yields a new room ID; normally I'd expect a `setup_*` future to be uninteresting.